packages/services/src/services/Translation/__tests__/Translation.test.js

```diff
@@ -76,7 +98,69 @@ describe('TranslationAPI', () => {
         origin: 'https://ibm.com',
       },
     });
-
+    delete response['id'];
     expect(response).toEqual(responseSuccess);
   });
+
+  it('should return a json with a recent timestamp', async () => {
+    const mockDate = 1546300800000; // Epoch time of January 1, 2019 midnight UTC
+    global.Date.now = jest.fn(() => mockDate);
+
+    // using very old cached session
+    sessionStorageMock.setItem(
+      'dds-translation-us-en',
+      JSON.stringify(oldSession)
+    );
+
+    const previousSession = JSON.parse(
+      sessionStorageMock.getItem('dds-translation-us-en')
+    );
+
+    expect(previousSession.id).toEqual('TRANSLATION_FRESH');
```

Comment:
Given we are not testing `sessionStorageMock` here, I don't think this is needed:

```suggestion
```